code/modules/fluids/fluid_core.dm

```diff
@@ -214,6 +214,28 @@ var/mutable_appearance/fluid_ma
 		var/turf/T = src.loc
 		T.Attackhand(user)
 
+	mouse_drop(obj/over_object, src_location, over_location)
+		if (istype(src, /obj/fluid/airborne) || !istype(over_object, /obj/item/reagent_containers/glass) && !istype(over_object, /obj/item/reagent_containers/food/drinks) && !istype(over_object, /obj/reagent_dispensers) && !istype(over_object, /obj/item/spraybottle) && !istype(over_object, /obj/machinery/plantpot) && !istype(over_object, /obj/mopbucket) && !istype(over_object, /obj/item/reagent_containers/mender) && !istype(over_object, /obj/item/tank/jetpack/backtank) && !istype(over_object, /obj/machinery/bathtub))
```

Comment:
any way to use `mousedrop_refill` here?

Reply:
The `istype` checks were pinched from the `mouse_drop` code for `/obj/machinery/bathtub`, and while using the `mousedrop_refill` list would be far cleanlier than a list of `istype` checks, I'm rather unsure as to how to correctly implement it.